lib/travis.js

```diff
@@ -20,7 +20,7 @@ const getTravisRedundantFeatures = async build => {
   return getRedundantFeaturesFromLog(prunerLog);
 };
 
-const doesBuildNotFailAllChecks = async buildId => {
+const doesBuildPassAllChecks = async buildId => {
   const { jobs } = await getJobsFromBuild(buildId);
   const failedJobs = jobs.filter(job => job.state !== 'passed');
```

Comment:
while non-blocking, a recent learning for myself is that lines 25-26 can be replaced by `jobs.some(job => job.state !== 'passed')`

Reply:
Is it actually equivalent to `jobs.every(job => job.state === 'passed')`? (Trying to avoid the double negative)